Spanner/src/Session/CacheSessionPool.php

```diff
@@ -650,18 +650,31 @@ private function getSession(array &$data)
      */
     private function createSessions($count)
     {
-        $sessions = [];
+        $args = [
+                'database' => $this->database->name(),
+                'session' => [
+                    'labels' => isset($this->config['labels']) ? $this->config['labels'] : []
+                ]
+            ];
+
+        $promises = [];
 
         for ($i = 0; $i < $count; $i++) {
-            $options = [];
-            if ($this->config['labels']) {
-                $options['labels'] = $this->config['labels'];
-            }
+            $promises[] = $this->database->connection()->createSessionAsync($args);
+        }
 
-            $sessions[] = [
-                'name' => $this->database->createSession($options)->name(),
-                'expiration' => $this->time() + SessionPoolInterface::SESSION_EXPIRATION_SECONDS
-            ];
+        $results = \GuzzleHttp\Promise\settle($promises)->wait();
```

Comment:
Minor nit here :), could we please import the namespace?

Reply:
👍 I'll import the Promise namespace.